### PR TITLE
add pi-wifi types

### PR DESCRIPTION
**Pull request by loofkid**
Please fill in this template.

- [* ] Use a meaningful title for the pull request. Include the name of the package modified.
- [* ] Test the change in your own code. (Compile and run.)
- [* ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [* ] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [* ] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [* ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [* ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Review by peterblazejewicz — CHANGES_REQUESTED** [review]
Please remove comments left from template
Those are comments to get someone started, not to be included in the code

File: types/pi-wifi/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,186 @@
+// Type definitions for pi-wifi 1.2
+// Project: https://github.com/matrix-io/pi-wifi
+// Definitions by: Kent Loofbourrow <https://github.com/loofkid>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace piWifi;
+
```
Comment (peterblazejewicz) [inline types/pi-wifi/index.d.ts:7]: this is when package is kind of UMD module:
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html#optional-global-usage
while this package looks like good, old CJS:
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-d-ts.html#common-commonjs-patterns
(if you change this, you would have to add proper import in tests)

Comment (loofkid) [reply types/pi-wifi/index.d.ts]: I believe I've fixed this. Thank you for helping me get this right, it's my first (official) type definition.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@loofkid thx!

**Comment by loofkid** [comment]
Ready to merge

**Pull request merged**